src/com/rcv/CVRReader.java

```diff
@@ -136,14 +136,14 @@ public void parseCVRFile(
           // empty cells are sometimes treated as undeclared write-ins (Portland / ES&S)
           if (config.isTreatBlankAsUWIEnabled()) {
             candidate = config.getUndeclaredWriteInLabel();
-            Logger.log("Empty cell -- treating as UWI");
+            Logger.warn("Empty cell -- treating as UWI");
```

Comment:
I think we should avoid acronyms when possible; recommend spelling out "undeclared write-in".

Reply:
moved this to #74